app/controllers/service_controller.rb

```diff
@@ -4,6 +4,12 @@ class ServiceController < ApplicationController
 
   helper_method :assignments_count
 
+  rescue_from DatastoreApi::Errors::ApiError do |e|
+    Rails.error.report(e, handled: false)
```

Comment:
I'm going to dig into this. I was thinking of how the error appears in the reporting service. I thought handled: true for when the error was not unexpected and not an error in the service and  handle: false, when it's something that should not have happened, but we've tidied up the way we show that to to the user. Will report back!

Reply:
I assume you're thinking about this atm before wanting a re-review @timpeat ? I agree with Jesus I think, as logically we are handling it. But idk the impact of that on the reporting

Reply:
Thanks @zheileman and @arthurashman . My investigation has been inconclusive. Sentry does care about the handled true|false in terms of how it notifies. However, within the application context handled: true does make more sense as it can effect how that error is then dealt with should it be raised again. 

For consistency, I have changed to handled: true as @zheileman suggests. But, I have also added severity :error as handled:true otherwise defaults to severity:warning.

Reply:
Nice! makes sense to me 👍